IronFoundry.Container.Shared/IronFoundry.Container.Shared.csproj

```diff
@@ -70,6 +73,21 @@
     <Compile Include="Messages\PingMessage.cs" />
     <Compile Include="Messages\ProcessDataEvent.cs" />
     <Compile Include="Messages\WaitForProcessExitMessage.cs" />
+    <Compile Include="Utilities\BackgroundProcess.cs" />
```

Comment:
I want to kill this. I think it's only used by `NetShRunner`.  We should rewrite that to use `IProcessRunner` instead.  We can add a TODO for this, but want to make sure we don't forget.